CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Release notes
 =============
 
+v1.0.12 2025-12-02
+------------------
+
+* Include the 2-Yukawa structure factor model
+* Add Ruff pre-commit hooks for developers
+* Figures are cached between builds for CI speed up
+* Plugin model location docs updated
```

Comment:
Truncated octahedran model.

This doc is for user-facing changes. I wouldn't mention developer changes in this doc (pre-commit hook and figure caching).